tests/test_bucketlist_api.py

```diff
@@ -0,0 +1,125 @@
+from unittest import TestCase
+import bucketlist
+import json
+
+
+class TestBucketlistAPI(TestCase):
+
+    def setUp(self):
+        self.client = bucketlist.app.test_client()
+        correct_credentials = json.dumps({"email":
+                                          "mark.nganga@andela.com",
+                                          "password": "p@ssw0rd"})
+        response = self.client.post('/auth/login',
+                                    data=correct_credentials,
+                                    content_type='application/json')
+
+        content = json.loads(response.get_data())
+        self.access_token = content['access_token']
+
+    def test_it_creates_bucketlist(self):
+        new_bucketlist = json.dumps({"list_title": "Third List", "list_description": "This is the decription"})
+
+        response = self.client.post('/bucketlists',
+                                    data=new_bucketlist,
+                                    content_type='application/json',
+                                    headers={'Authorization': 'JWT %s' % self.access_token})
+
+        content = json.loads(response.get_data(as_text=True))
+        self.list_id = content['bucketlist']['list_id']
+        self.assertEqual(response.status_code, 201)
+
+    def test_it_updates_bucketlist(self):
+        new_bucketlist = json.dumps({"list_title": "Third List", "list_description": "This is the decription"})
+
+        response = self.client.post('/bucketlists',
+                                    data=new_bucketlist,
+                                    content_type='application/json',
+                                    headers={'Authorization': 'JWT %s' % self.access_token})
+
+        content = json.loads(response.get_data(as_text=True))
+        self.list_id = content['bucketlist']['list_id']
+
+
+        new_bucketlist = json.dumps({"list_title": "Third List Updated", "list_description": "This is the updated decription"})
+
+        response = self.client.put('/bucketlists/%s' % self.list_id,
+                                    data=new_bucketlist,
+                                    content_type='application/json',
+                                    headers={'Authorization': 'JWT %s' % self.access_token})
+
+        content = json.loads(response.get_data(as_text=True))
+        self.assertEqual(response.status_code, 200)
```

Comment:
the response code should be 201 consider reading about the different HTTP status codes and what they mean